demosplan/DemosPlanCoreBundle/Entity/Procedure/Procedure.php

```diff
@@ -2107,37 +2096,21 @@ public function setStatements(ArrayCollection $statements): void
 
     public function getSurveys(): Collection
     {
-        return $this->surveys;
+        return new ArrayCollection();
     }
 
-    /**
-     * Returns first Survey in the list.
-     *
-     * @param string $surveyId
-     */
-    public function getSurvey($surveyId): ?Survey
+    public function getSurvey($surveyId): ?SurveyInterface
```

Comment:
shouldn't this be deleted entirely?
I can't find any calls.

Reply:
@salisdemos  I wanted to keep the interface in this PR and clean up the Interface in a next PR. Should have mentioned it in the description